e2e/clone_test.go

```diff
@@ -0,0 +1,229 @@
+package e2e
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+
+	//	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	//go:embed testdata/cloning/clone-pod-template.yaml
+	thinPodCloneTemplateYAML string
+
+	//go:embed testdata/cloning/clone-pvc-template.yaml
+	thinPvcCloneTemplateYAML string
+)
+
+const (
+	thinClonePVCName = "thin-clone"
+	thinClonePodName = "thin-clone-pod"
+)
+
+func testPVCClone() {
+	nsCloneTest := "clone-test" + randomString(10)
```

Comment:
This is usually generated in BeforeEach() so there is a different ns for each test.

Reply:
True, the namespace is created in BeforeEach: https://github.com/topolvm/topolvm/pull/546/files#diff-d835c055cb3eb10ddad0963ec6a9afd91f3c58b1ed29dd59d232a1b2a8a84be0R32